index.html

```diff
@@ -1338,6 +1336,53 @@ <h3>alsoKnownAs</h3>
         </div>
 
       </section>
+
+      <section>
+        <h3>deterministicSameAs</h3>
```

Comment:
@csuwildcat -- The spec entered feature freeze on July 15th 2020. 

I'm trying to figure out which issue or PR this is a continuation of in order to defend the addition of this language to the specification. If we can't point to an active issue or PR, then we can't add this feature (as it'll be considered a new feature).

This feature also seems SideTree specific... are there any other DID Methods that need this feature?

To be clear, you can always add this feature in another spec and add it to the registry... that may be an easier path than trying to convince the WG that the feature is general enough to go into DID Core.

Reply:
@msporny We need the canonical DID capability in the trustbloc method for two reasons:
- Late publishing (this is from our reliance on the Sidetree protocol spec, but seems like a generic concept).
- Network segment switching in the DID (https://github.com/w3c/did-core/issues/421#issuecomment-701671659). This capability is unrelated to our reliance on the Sidetree protocol spec.

To accomplish the concept of a canonical DID (in a generic manner), we need:
- an assertion of equivalence; and
- we need to signal that the RP should use the other DID going forward.

Reply:
Sidetree isn't a method, it's a meta-protocol. The methods that would benefit are:

- ION 
- Element
- IOHK's
- trustbloc
- DID Peer
- methods that use KERI

This is related to all the canonical deterministic equivalence and past discussions of the need that span almost 2 years. (remember that thing with Joe where we needed to have the chairs mediate a call? That was all surrounding this, and it's been an unaddressed need for a long time)

Reply:
The first official thread, besides all the call discussion that preceded it for over a year before that, is here: https://github.com/w3c/did-core/issues/70

^ without a good solution to this, we're making what will probably be a massive portion of the total DIDs floating around less usable in critical scenarios.

Reply:
I think the semantics of this property are sufficiently different from `alsoKnownAs`, but I like the particular name `deterministicSameAs` less than some of the other ones that were proposed here: https://github.com/w3c/did-core/issues/421